Specs/Core/QuaternionSpec.js

```diff
@@ -652,7 +660,7 @@ defineSuite([
             Quaternion.fromRotationMatrix(undefined);
         }).toThrowDeveloperError();
     });
-
+/*
```

Comment:
As fromHeadingPitchRoll use now a headingPitchRoll class, the commented tests have no interest.